redash/query_runner/athena.py

```diff
@@ -1,102 +1,127 @@
+import logging
 import json
-import os
 
-import requests
+from redash.utils import JSONEncoder
+from redash.query_runner import *
 
-from redash.query_runner import BaseQueryRunner, register
-from redash.settings import parse_boolean
 
-PROXY_URL = os.environ.get('ATHENA_PROXY_URL')
-ANNOTATE_QUERY = parse_boolean(os.environ.get('ATHENA_ANNOTATE_QUERY', 'true'))
+logger = logging.getLogger(__name__)
+
+
+try:
+    import pyathena
+    enabled = True
+except ImportError:
+    enabled = False
+
+
+_TYPE_MAPPINGS = {
+    'boolean': TYPE_BOOLEAN,
+    'tinyint': TYPE_INTEGER,
+    'smallint': TYPE_INTEGER,
+    'integer': TYPE_INTEGER,
+    'bigint': TYPE_INTEGER,
+    'double': TYPE_FLOAT,
+    'varchar': TYPE_STRING,
+    'timestamp': TYPE_DATETIME,
+    'date': TYPE_DATE,
+    'varbinary': TYPE_STRING,
+    'array': TYPE_STRING,
+    'map': TYPE_STRING,
+    'row': TYPE_STRING,
+    'decimal': TYPE_FLOAT,
+}
+
 
 class Athena(BaseQueryRunner):
     noop_query = 'SELECT 1'
 
     @classmethod
-    def name(cls):
-        return "Amazon Athena"
-
-    @classmethod
     def configuration_schema(cls):
         return {
             'type': 'object',
             'properties': {
-                'region': {
+                'aws_access_key_id': {
                     'type': 'string',
-                    'title': 'AWS Region'
+                    'title': 'AWS Access Key ID',
                 },
-                'aws_access_key': {
+                'aws_secret_access_key': {
                     'type': 'string',
-                    'title': 'AWS Access Key'
+                    'title': 'AWS Secret Access Key',
                 },
-                'aws_secret_key': {
+                'schema_name': {
                     'type': 'string',
```

Comment:
Let's add a default value here (`default`) and then it can be no longer a mandatory field.